doc/widgets/kmeans.rst

```diff
@@ -1,86 +1,105 @@
 Interactive k-means
 ===================
 
-.. figure:: icons/mywidget.png
+.. figure:: icons/interactive-kmeans.png
 
-Educational widgets that shows working of a k-means clustering.
+Educational widget that shows the working of a k-means clustering.
 
 Signals
 -------
+
 Inputs
 ~~~~~~
-- **Data**
+
+-  **Data**
+
 Input data set.
 
-Outputs
-~~~~~~~
+**Outputs**:
```

Comment:
Agh, yes. There's a bit of a difference between Orange3 documentation style and add-ons doc style. It really depends on the markdown. It bugs me a bit that we have two different styles in our docs.... what's your opinion on this?